hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/snapshot/OMSnapshotPurgeRequest.java

```diff
@@ -125,11 +135,23 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, Execut
       omSnapshotIntMetrics.incNumSnapshotPurges();
       LOG.info("Successfully executed snapshotPurgeRequest: {{}} along with updating snapshots:{}.",
           snapshotPurgeRequest, updatedSnapshotInfos);
+      
+      Map<String, String> auditParams = new HashMap<>();
```

Comment:
a common method to prepare audit map, with both failure/success, as failure will have additional parameter of failure information. follow similar mechansim as done for other audit.